### DISCUSSION TITLE
The Best of two Breeds

### DISCUSSION THREAD

**Discussion by mbarnig** [coqui-ai/TTS#594] [Show and tell]
The inclusion of [rhasspy/gruut](https://github.com/rhasspy/gruut) into [Coqui-TTS](https://github.com/coqui-ai/TTS/tree/dev) today (dev branch) is a milestone to achieve the goal "providing open speech technologies for everyone". A few days ago I installed the *add-gruut branch* of the [forked Coqui-TTS](https://github.com/rhasspy/TTS) Github repository on my Desktop-PC to train a voice with a small french dataset. I posted [my findings](https://www.web3.lu/the-best-of-two-breeds/) in the blog of my website "Internet with a Brain". Thanks to @erogol and @synesthesiam who made this possible.

**Comment by erogol** [comment]
I guess the link to "your findings" is wrong :) 
And thx for your kind words !

**Reply by mbarnig** [reply]
You are right. I fixed it. Thanks.

**Comment by synesthesiam** [comment]
Thanks, @mbarnig! ([fixed link to your findings](https://www.web3.lu/the-best-of-two-breeds/))

And thanks to @erogol for giving gruut a chance :slightly_smiling_face: 
I'm hoping to contribute more to :frog: TTS as time goes on, such as training more freely-available models and assisting with Onnx export.
